### PR TITLE
[SD-3427] - Default values for urgency and priorty

### PR DESCRIPTION
**Pull request by akintolga**

### CODE REVIEW

**Comment by sivakuna-aap** [comment]
:+1: 
Quick Question: Should we override `urgency and priority` always on article creation? Shouldn't it have same values what were given in the template if the article is being created from template?

**Comment by akintolga** [comment]
Thanks @sivakuna-aap. Now it uses the default values if they're empty.

File: server/apps/archive/archive.py [hunk 1]
```diff
@@ -165,6 +167,12 @@ def on_create(self, docs):
             if not doc.get('ingest_provider'):
                 doc['source'] = DEFAULT_SOURCE_VALUE_FOR_MANUAL_ARTICLES
 
+            if 'urgency' not in doc:
+                doc['urgency'] = DEFAULT_URGENCY_VALUE_FOR_MANUAL_ARTICLES
+
+            if 'priority' not in doc:
+                doc['priority'] = DEFAULT_PRIORITY_VALUE_FOR_MANUAL_ARTICLES
```
Comment (plamut) [inline server/apps/archive/archive.py]: Tip: the same can be achieved in a more concise way by using the `dict`'s [setdefault()](https://docs.python.org/3.4/library/stdtypes.html#dict.setdefault) method. Example:
```py
doc.setdefault('priority', DEFAULT_PRIORITY_VALUE_FOR_MANUAL_ARTICLES)
```

**Comment by plamut** [comment]
Apart from a code reafactoring tip that will make you look more sophisticated (:laughing:) it looks good.

:+1:

**Comment by mdhaman** [comment]
:+1:

**Comment by amagdas** [comment]
:+1:

**Pull request merged**